ERROR - 2024-06-08 07:49:49 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 07:49:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 07:49:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:49:49\', \'2024-06-08 07:49:49\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:49:49\', \'2024-06-08 07:49:49\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 07:49:49 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:49:49\', \'2024-06-08 07:49:49\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:49:49\', \'2024-06-08 07:49:49\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:49:49\', \'2024-06-08 07:49:49\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
ERROR - 2024-06-08 07:55:49 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 07:55:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 07:55:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:55:49\', \'2024-06-08 07:55:49\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:55:49\', \'2024-06-08 07:55:49\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 07:55:49 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:55:49\', \'2024-06-08 07:55:49\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:55:49\', \'2024-06-08 07:55:49\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:55:49\', \'2024-06-08 07:55:49\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
ERROR - 2024-06-08 07:56:05 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 07:56:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 07:56:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:56:05\', \'2024-06-08 07:56:05\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:56:05\', \'2024-06-08 07:56:05\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 07:56:05 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:56:05\', \'2024-06-08 07:56:05\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:56:05\', \'2024-06-08 07:56:05\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aaa\', \'2024-06-08 07:56:05\', \'2024-06-08 07:56:05\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
ERROR - 2024-06-08 07:58:57 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 07:58:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 07:58:57 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 07:58:57\', \'2024-06-08 07:58:57\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 07:58:57\', \'2024-06-08 07:58:57\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 07:58:57 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 07:58:57\', \'2024-06-08 07:58:57\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 07:58:57\', \'2024-06-08 07:58:57\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 07:58:57\', \'2024-06-08 07:58:57\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
ERROR - 2024-06-08 08:01:54 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 08:01:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:01:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:54\', \'2024-06-08 08:01:54\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:54\', \'2024-06-08 08:01:54\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:01:54 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:54\', \'2024-06-08 08:01:54\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:54\', \'2024-06-08 08:01:54\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:54\', \'2024-06-08 08:01:54\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
ERROR - 2024-06-08 08:01:57 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 08:01:57 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:01:57 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:57\', \'2024-06-08 08:01:57\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:57\', \'2024-06-08 08:01:57\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:01:58 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:57\', \'2024-06-08 08:01:57\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:57\', \'2024-06-08 08:01:57\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:01:57\', \'2024-06-08 08:01:57\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
INFO - 2024-06-08 08:02:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-08 08:02:12 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 08:02:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:02:12 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:02:12\', \'2024-06-08 08:02:12\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:02:12\', \'2024-06-08 08:02:12\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:02:12 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:02:12\', \'2024-06-08 08:02:12\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:02:12\', \'2024-06-08 08:02:12\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 08:02:12\', \'2024-06-08 08:02:12\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
INFO - 2024-06-08 08:07:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-08 08:07:28 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 08:07:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:07:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 08:07:28\', \'2024-06-08 08:07:28\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 08:07:28\', \'2024-06-08 08:07:28\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:07:28 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 08:07:28\', \'2024-06-08 08:07:28\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 08:07:28\', \'2024-06-08 08:07:28\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 08:07:28\', \'2024-06-08 08:07:28\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
INFO - 2024-06-08 08:58:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-08 08:58:28 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\LoginController.php(21): CodeIgniter\BaseModel->first()
#7 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\LoginController->login()
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#14 {main}
CRITICAL - 2024-06-08 08:58:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\LoginController.php(21): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\LoginController->login()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:58:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `email` = \'Email@email.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `email` = \'Email@email.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\LoginController.php(21): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\LoginController->login()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 08:58:28 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `users`
WHERE `email` = \'Email@email.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `email` = \'Email@email.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `email` = \'Email@email.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\LoginController.php(21): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\LoginController->login()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\LoginController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
INFO - 2024-06-08 08:58:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 08:58:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 13:53:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-08 13:53:34 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 13:53:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 13:53:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:53:34\', \'2024-06-08 13:53:34\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:53:34\', \'2024-06-08 13:53:34\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 13:53:34 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:53:34\', \'2024-06-08 13:53:34\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:53:34\', \'2024-06-08 13:53:34\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:53:34\', \'2024-06-08 13:53:34\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
ERROR - 2024-06-08 13:55:38 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 13:55:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 13:55:38 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:37\', \'2024-06-08 13:55:37\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:37\', \'2024-06-08 13:55:37\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 13:55:38 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:37\', \'2024-06-08 13:55:37\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:37\', \'2024-06-08 13:55:37\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'2210817210032\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:37\', \'2024-06-08 13:55:37\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
INFO - 2024-06-08 13:55:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-08 13:55:51 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `us...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(32): CodeIgniter\Model->insert(Array)
#8 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#15 {main}
CRITICAL - 2024-06-08 13:55:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 13:55:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:51\', \'2024-06-08 13:55:51\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:51\', \'2024-06-08 13:55:51\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 13:55:51 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:51\', \'2024-06-08 13:55:51\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:51\', \'2024-06-08 13:55:51\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (\'indrawijaya34\', \'Email@email.com\', \'aa\', \'2024-06-08 13:55:51\', \'2024-06-08 13:55:51\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `user` (`username`, `email`, `password`, `created_at`, `updated_at`) VALUES (:username:, :email:, :password:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RegisterController.php(32): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
INFO - 2024-06-08 13:59:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:00:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:00:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:00:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:00:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:00:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:00:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:00:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:00:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:01:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:01:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:11:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-06-08 14:11:24 --> mysqli_sql_exception: Table 'modul6.users' doesn't exist in D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1\nFROM `...', 0)
#1 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('indrawijaya1501...', 'email', Array, NULL, 'email')
#6 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'indrawijaya1501...', Array, Array, 'email')
#7 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 D:\laragon\www\LAPORANPRAKTIKUM6\app\Controllers\RegisterController.php(38): CodeIgniter\Controller->validate(Array)
#9 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\RegisterController->store()
#10 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
#11 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#13 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 D:\laragon\www\LAPORANPRAKTIKUM6\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 D:\laragon\www\LAPORANPRAKTIKUM6\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\...')
#16 {main}
CRITICAL - 2024-06-08 14:11:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
[Method: POST, Route: register]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('indrawijaya1501@gmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'indrawijaya1501@gmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\RegisterController.php(38): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 14:11:24 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'indrawijaya1501@gmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'indrawijaya1501@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('indrawijaya1501@gmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'indrawijaya1501@gmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\RegisterController.php(38): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
CRITICAL - 2024-06-08 14:11:24 --> [Caused by] mysqli_sql_exception: Table 'modul6.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT 1
FROM `users`
WHERE `email` = \'indrawijaya1501@gmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `users`
WHERE `email` = \'indrawijaya1501@gmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `users`
WHERE `email` = \'indrawijaya1501@gmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `users`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('indrawijaya1501@gmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'indrawijaya1501@gmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\RegisterController.php(38): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RegisterController->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RegisterController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\Commands\Server\rewrite.php(49): require_once('D:\\laragon\\www\\LAPORANPRAKTIKUM6\\public\\index.php')
INFO - 2024-06-08 14:12:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:12:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:12:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:12:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:12:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:12:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:12:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:13:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:13:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:13:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:13:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:13:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:14:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:14:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:14:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:14:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:15:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:15:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:15:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:15:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:15:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:15:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:17:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:17:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:17:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:17:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:17:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:17:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:20:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:20:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:20:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:20:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:25:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:26:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:26:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:26:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:26:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:26:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:26:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:26:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:33:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:33:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:33:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2024-06-08 14:34:01 --> Email: 2210817210032@mhs.ulm.ac.id
DEBUG - 2024-06-08 14:34:01 --> Password: 12345678
INFO - 2024-06-08 14:34:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-06-08 14:35:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2024-06-08 14:35:50 --> Email: 2210817210032@mhs.ulm.ac.id
DEBUG - 2024-06-08 14:35:50 --> Password: 12345678
INFO - 2024-06-08 14:35:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2024-06-08 14:35:52 --> Email: 2210817210032@mhs.ulm.ac.id
DEBUG - 2024-06-08 14:35:52 --> Password: 12345678
INFO - 2024-06-08 14:35:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
